website/content/en/docs/kubebuilder/operator-scope.md

```diff
@@ -0,0 +1,288 @@
+# Operator Scope
+
+## Overview
+
+A `namespace-scoped` operator watches and manages resources in a single namespace, whereas a `cluster-scoped` operator watches and manages resources cluster-wide. 
```

Comment:
The *-scoped formatting doesn't make sense, since those terms are not formal objects:
```suggestion
A namespace-scoped operator watches and manages resources in a single namespace, whereas a cluster-scoped operator watches and manages resources cluster-wide. 
```